platform/android/src/file_source.hpp

```diff
@@ -28,6 +29,14 @@ class FileSource {
         static std::string onURL(jni::JNIEnv&, const jni::Object<FileSource::ResourceTransformCallback>&, int, std::string);
     };
 
+    struct ResourcesCachePathChangeCallback {
+        static constexpr auto Name() { return "com/mapbox/mapboxsdk/storage/FileSource$ResourcesCachePathChangeCallback";}
+
+        static void onSuccess(jni::JNIEnv&,
+                              const jni::Object<FileSource::ResourcesCachePathChangeCallback>&,
+                              std::string);
```

Comment:
maybe change to `const jni::String&` and convert std::string to jni::string before calling `onSuccess`